lib/coins/btc.js

```diff
@@ -17,7 +17,7 @@ const bech32Opts = {
 module.exports = {depositUrl, parseUrl, formatAddress, base58Opts, bech32Opts, createWallet}
 
 function parseUrl (network, url) {
-  const res = /^([bB]itcoin:\/{0,2})?(\w+)/.exec(url)
+  const res = /^(bitcoin:\/{0,2})?(\w+)/i.exec(url)
```

Comment:
check if it's necessary to allow between 0 and 2 forward slashes between the first capture group and the second